community-operators/hazelcast-enterprise/hazelcast-operator.v0.0.1.clusterserviceversion.yaml

```diff
@@ -0,0 +1,208 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  annotations:
+    capabilities: Basic Install
+    alm-examples: |-
+      [{
+       "apiVersion": "hazelcast.com/v1alpha1",
+       "kind": "Hazelcast",
+       "metadata": {
+          "name": "hz"
+       },
+       "spec": {
+          "image": {
+            "tag": "3.11.2"
+          },
+          "hazelcast": {
+            "licenseKeySecretName": "hz-license-key-secret"
+          },
+          "serviceAccount": {
+            "create": false,
+            "name": "hazelcast"
+          }
+       }
+      }]
+    categories: "Database"
+    certified: "false"
+    containerImage: hazelcast/hazelcast-enterprise-operator:0.0.1
+    createdAt: 2019-03-14T09:58:52Z
+    description: Install Hazelcast Enterprise cluster.
+    support: Hazelcast, Inc
+  name: hazelcast-operator.v0.0.1
+  namespace: placeholder
+spec:
+  apiservicedefinitions: {}
```

Comment:
nit: Please remove this field since it's empty.

Reply:
Done